server/src/main/java/org/opentosca/toscana/core/parse/CsarParseServiceImpl.java

```diff
@@ -6,18 +6,20 @@
 
 import org.opentosca.toscana.core.csar.Csar;
 import org.opentosca.toscana.core.csar.CsarDao;
+import org.opentosca.toscana.core.parse.converter.ModelConverter;
 import org.opentosca.toscana.model.EffectiveModel;
 
 import org.eclipse.winery.model.tosca.yaml.TServiceTemplate;
 import org.eclipse.winery.yaml.common.reader.yaml.Reader;
 import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 
 @Service
 public class CsarParseServiceImpl implements CsarParseService {
 
-    private Logger logger;
+    private Logger logger = LoggerFactory.getLogger(CsarParseService.class);
```

Comment:
I think we had the intention of sending the logs to the client if the parsing failed!

Reply:
We will. I will add documentation that explains why this happens.
- as you see in the method beneath, the logger is assigned to the csar's logger.
- this LoggerFactory.getLogger() assignment is actually only needed for testing purposes:
It's possible to call the (also public) method `parse(File)` (which is used internally in `parse(Csar)` for testing purposes. In this case, `logger` would be null.